resizeobserver/demo.js

```diff
@@ -0,0 +1,13 @@
+/* global ResizeObserver */
+
+const ro = new ResizeObserver(entries => {
```

Comment:
This code throws a `ResizeObserver loop limit exceeded` error as soon as the page is opened. (Testing on Chrome 55.0.2883.0 with the "Experimental Web Platform Features" flag enabled.)

![image](https://cloud.githubusercontent.com/assets/1749548/19205290/7a923d46-8caf-11e6-82f5-925d33ce796d.png)